### PR TITLE
Css handlers

### PR DESCRIPTION
**Pull request by khrizzcristian**
Added 7 more css handlers

### CODE REVIEW

**Comment by khrizzcristian** [comment]
What is wrong with my pull request? I can't find any issue in my code but I found in Lint this kind of errors

##[error]  496:18  error    Replace `·className={`${handles.reviewsRatingAverage}·review__rating--average·dib·v-mid`}` with `⏎··············className={`${handles.reviewsRatingAverage}·review__rating--average·dib·v-mid`}⏎············`                                                                prettier/prettier
##[error]  504:18  error    Replace `·className={`${handles.reviewsRatingCount}·review__rating--count·dib·v-mid`}` with `⏎··············className={`${handles.reviewsRatingCount}·review__rating--count·dib·v-mid`}⏎············`                                                                        prettier/prettier
##[error]  597:27  error    Replace `·className={`${handles.reviewCommentRating}·review__comment--rating·t-heading-5`}` with `⏎························className={`${handles.reviewCommentRating}·review__comment--rating·t-heading-5`}⏎······················`                                          prettier/prettier
##[error]  599:30  error    Replace `·className={`${handles.reviewCommentUser}·review__comment--user·lh-copy·mw9·t-heading-5·mt0·mb2`}` with `⏎··························className={`${handles.reviewCommentUser}·review__comment--user·lh-copy·mw9·t-heading-5·mt0·mb2`}⏎························`      prettier/prettier
##[error]  639:29  error    Replace `·className={`${handles.reviewCommentRating}·review__comment--rating·t-heading-5`}` with `⏎··························className={`${handles.reviewCommentRating}·review__comment--rating·t-heading-5`}⏎························`                                      prettier/prettier
##[error]  641:32  error    Replace `·className={`${handles.reviewCommentUser}·review__comment--user·lh-copy·mw9·t-heading-5·mt0·mb2`}` with `⏎····························className={`${handles.reviewCommentUser}·review__comment--user·lh-copy·mw9·t-heading-5·mt0·mb2`}⏎··························`  prettier/prettier

File: docs/README.md [hunk 1]
```diff
@@ -68,18 +68,18 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+| 'loginLink'               |
+| 'reviewsRating'           |
+| 'starsContainer'          |
+| 'reviewsHeading'          |
+| 'reviewsRatingAverage'    |
+| 'reviewsRatingCount'      |
+| 'reviewCommentsContainer' |
+| 'reviewComment'           |
+| 'reviewCommentRating'     |
+| 'reviewCommentUser'       |
```
Comment (klzns) [inline docs/README.md:82]: It's better to keep the list sorted and with code formatting. Also some existing CSS Handles are missing.

```suggestion
| `container`               |
| `loginLink`               |
| `reviewComment`           |
| `reviewCommentRating`     |
| `reviewCommentsContainer` |
| `reviewCommentUser`       |
| `reviewsHeading`          |
| `reviewsRating`           |
| `reviewsRatingAverage`    |
| `reviewsRatingCount`      |
| `star--empty`             |
| `star--filled`            |
| `star`                    |
| `starpicker`              |
| `starsContainer`          |
| `writeReviewContainer`    |
```

File: react/Reviews.tsx [hunk 2]
```diff
@@ -297,8 +297,15 @@ const CSS_HANDLES = [
...
   'loginLink',
+  'reviewsRating',
+  'starsContainer',
   'reviewsHeading',
+  'reviewsRatingAverage',
+  'reviewsRatingCount',
   'reviewCommentsContainer',
+  'reviewComment',
+  'reviewCommentRating',
+  'reviewCommentUser'
```
Comment (klzns) [inline react/Reviews.tsx:308]: Missing trailing comma `,`

File: react/Reviews.tsx [hunk 3]
```diff
@@ -478,23 +485,23 @@ const Reviews: FunctionComponent<InjectedIntlProps & Props> = props => {
...
         {!state.hasTotal || !state.hasAverage ? (
           <FormattedMessage id="store/reviews.list.summary.loading" />
         ) : !state.total ? null : (
           <Fragment>
-            <div className="t-heading-4">
+            <div className={`${handles.starsContainer} t-heading-4`}>
               <Stars rating={state.average} />
             </div>
-            <span className="review__rating--average dib v-mid">
+            <span className={`${handles.reviewsRatingAverage} review__rating--average dib v-mid`}>
```
Comment (klzns) [inline react/Reviews.tsx:496]: Lint issue here (reported by the GitHub Action)

To setup the linter you should run:
-  `yarn` on the root directory
-  `yarn` on the `react` directory

Then run `yarn lint --fix` on the root directory to fix the issues.

**Review by klzns — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (guerreirobeatriz) [inline docs/README.md:82]:
```suggestion
| 'container'               |
| 'loginLink'               |
| 'reviewComment'           |
| 'reviewCommentsContainer' |
| 'reviewCommentRating'     |
| 'reviewCommentUser'       |
| 'reviewsHeading'          |
| 'reviewsRating'           |
| 'reviewsRatingAverage'    |
| 'reviewsRatingCount'      |
| 'starsContainer'          |
| 'writeReviewContainer'    |
```

**Review by guerreirobeatriz — CHANGES_REQUESTED** [review]

**Comment by khrizzcristian** [comment]
How can I fix that Danger CI error?

File: docs/README.md [hunk 4]
```diff
@@ -69,16 +69,20 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | CSS Handles               |
 | ------------------------- |
 | `container`               |
```
Comment (klzns) [inline docs/README.md:71]:
```suggestion
| `container`               |
| `formContainer`           |
```

File: docs/README.md [hunk 5]
```diff
@@ -69,16 +69,20 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+| `reviewsRating`           |
+| `reviewsRatingAverage`    |
+| `reviewsRatingCount`      |
 | `star--empty`             |
 | `star--filled`            |
 | `star`                    |
 | `starpicker`              |
-| `stars`                   |
-| `summaryContainer`        |
+| `starsContainer`          |
```
Comment (klzns) [inline docs/README.md:85]:
```suggestion
| `stars`                   |
| `starsContainer`          |
| `summaryContainer`        |
```

**Review by klzns — CHANGES_REQUESTED** [review]
The Danger CI is complaining about a missing entry in the CHANGELOG.md

You can add something like this under the [Unreleased] section of the file:

```diff
 ## [Unreleased]

+### Added
+- Missing CSS Handles
```

**Review by klzns — APPROVED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Comment by khrizzcristian** [comment]
Hello. Cam anyone metge my pull request?
Thank you

**Comment by klzns** [comment]
@allcontributors add @khrizzcristian for code

**Comment by khrizzcristian** [comment]
Hello. I can't deploy the changes
10:07:16.905 - error: Role User:cristian.longo@customsoft.ro in vtex/
cannot perform action PATCH on resource
vrn:apps:aws-us-east-1:vtex:master:/registry/vtex.reviews-and-ratings/2.1.0

*------------------*
*Cristian Longo*
*Web Developer*

**Pull request merged**